enlarge ui and add suggestions

Abab-bk opened the pull request:
enlarge ui and add suggestions.
edit: add paused when open console.

Abab-bk committed:
enlarge ui and add suggestions

LauraWebdev commented on demo/SofiaConsole.csproj.old:
This file can be removed if the project works properly.

LauraWebdev commented on demo/addons/sofiaconsole/Console.cs:
```diff
@@ -36,14 +40,20 @@ public override void _EnterTree()
         
         GD.Print("[SofiaConsole] Done");
+
+        // _commandInput.TextChanged += _ => UpdateUi();
```
Should be removed if unused.

LauraWebdev commented on demo/addons/sofiaconsole/Console.cs:
```diff
@@ -65,20 +75,56 @@ public override void _Input(InputEvent @event)
 
             // Press Up to toggle between previous commands
-            if (eventKey.Keycode == Key.Up && _commandInput.HasFocus() && _commandHistory.Count > 0)
+            // TODO
```
Should be removed if unused.

LauraWebdev commented on demo/addons/sofiaconsole/Console.cs:
```diff
@@ -65,20 +75,56 @@ public override void _Input(InputEvent @event)
+                if (_suggestions.Count > 0 && _currentSuggestionIndex >= 0)
                 {
-                    historyIndex = _commandHistory.Count;
+                    // _commandInput.Text = _suggestions[_currentSuggestionIndex];
```
Should be removed if unused.

LauraWebdev commented on demo/addons/sofiaconsole/Console.cs:
```diff
@@ -88,6 +134,45 @@ public override void _Input(InputEvent @event)
+
+            if (_currentSuggestionIndex >= 0 && _suggestions.Count > 0 && _currentSuggestionIndex < _suggestions.Count && _suggestions[_currentSuggestionIndex] == text)
+            {
+                newLabel.AddThemeStyleboxOverride("normal", GD.Load<StyleBoxFlat>("res://addons/sofiaconsole/StyleBox.tres"));
```
We should probably give this asset a proper name, StyleBox does not indicate what it is used for.

Abab-bk committed:
Refactor code for improved performance and optimization

Abab-bk committed:
Improve Ui and del demo/SofiaConsole.csproj.old

Abab-bk commented:
@LauraWebdev 
Updated, check plz.
I have reworked the messy UpdateUi() method and now it has become much better.
If there are any shortcomings, please point them out as I am a beginner in C#.
